### DISCUSSION TITLE
posts/Go/go_rabbitmq_tutorials_01/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#325] [Announcements]
https://www.liwenzhou.com/posts/Go/go_rabbitmq_tutorials_01/ 

 李文周的Blog rabbitmq 消息队列 Go语言 客户端 教程 amqp

**Comment by zhouhaoJOKER** [comment]
消费者 接受里面的channel 没有获取到值，不会导致死锁吗？

**Comment by Q1mi** [comment]
> @zhouhaoJOKER
> 消费者 接受里面的channel 没有获取到值，不会导致死锁吗？

正常的业务场景下没有消息队列中没有消息的时候会等待。

**Comment by zhouhaoJOKER** [comment]
李大哥。我是单纯看这个代码。好像没有给这个通道写入职。所以觉得会死锁😃。刚开始学

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: 李文周 <notifications@github.com>
发送时间: Thursday, December 17, 2020 8:24:54 PM
收件人: Q1mi/BlogComments <BlogComments@noreply.github.com>
抄送: zhouhao <Joker_Chou@outlook.com>; Mention <mention@noreply.github.com>
主题: Re: [Q1mi/BlogComments] RabbitMQ Go语言客户端教程1——HelloWorld | 李文周的博客 (#126)


@zhouhaoJOKER<https://github.com/zhouhaoJOKER>
消费者 接受里面的channel 没有获取到值，不会导致死锁吗？

正常的业务场景下没有消息队列中没有消息的时候会等待。

**Comment by zhoutonglx** [comment]
/images/Go/rabbitmq/tutorials01/sending.png 这里乱码了

**Comment by Q1mi** [comment]
> @zhoutonglx
> /images/Go/rabbitmq/tutorials01/sending.png 这里乱码了

感谢反馈，已修正。

**Comment by zhoutonglx** [comment]
> @Q1mi
> > @zhoutonglx
> > /images/Go/rabbitmq/tutorials01/sending.png 这里乱码了
> 
> 感谢反馈，已修正。

后面又乱码了了😂，像单词mandatory 这里

**Comment by Q1mi** [comment]
> @zhoutonglx
> > @Q1mi
> > > @zhoutonglx
> > > /images/Go/rabbitmq/tutorials01/sending.png 这里乱码了
> > 
> > 感谢反馈，已修正。
> 
> 后面又乱码了了😂，像单词mandatory 这里

😅 这次真的好了。。。
